lib/widgets/screens/profile/profile_basic_info.dart

```diff
@@ -0,0 +1,231 @@
+            ),
+          ],
+        ),
+    ]),
+    if (linkedinUrl != null)
+      Ink(
+        decoration: ShapeDecoration(
+          shape: RoundedRectangleBorder(
+              borderRadius:
+                  BorderRadius.circular(Radii.roundedRectRadiusSmall)),
+          color: theme.colorScheme.onInverseSurface,
+        ),
+        child: IconButton(
+          style: IconButton.styleFrom(
+              shape: RoundedRectangleBorder(
+            borderRadius: BorderRadius.circular(Radii.roundedRectRadiusSmall),
+          )),
+          icon: const ColorFiltered(
+            colorFilter: ColorFilter.mode(
+              Colors.white,
+              BlendMode.saturation,
+            ),
+            child: SizedBox(
+              width: _inlineIconSize,
+              child: Image(
+                image: AssetImage(Assets.linkedInIcon),
+              ),
+            ),
+          ),
+          onPressed: () {
+            debugPrint(linkedinUrl);
+            _launchUrl(Uri.parse(linkedinUrl));
+          },
+        ),
+      ),
+  ]);
+}
+
+Widget _createNameAndBadges(BuildContext context, String fullName,
+    String? pronouns, UserType userType, List<String>? affiliations) {
+  final AppLocalizations l10n = AppLocalizations.of(context)!;
+  final ThemeData theme = Theme.of(context);
+
+  List<Widget> widgets = [
+    Text(
+      fullName,
+      style: theme.textTheme.titleMedium?.copyWith(
+        color: theme.colorScheme.onSurface,
+      ),
+    ),
+    if (pronouns != null)
+      Text(pronouns,
+          style: theme.textTheme.bodySmall
+              ?.copyWith(color: theme.colorScheme.secondary)),
+    if (userType == UserType.mentor)
+      ProfileChip(
+        text: l10n.mentorCapitalized,
+        usePrimaryColor: true,
+      ),
+    if (userType == UserType.entrepreneur)
+      ProfileChip(
+        text: l10n.entrepreneurCapitalized,
+        usePrimaryColor: true,
+      ),
+  ];
+
+  if (affiliations != null) {
+    for (String affiliation in affiliations) {
+      widgets.add(ProfileChip(
+        text: affiliation,
+        usePrimaryColor: true,
+      ));
+    }
+  }
+  return Expanded(
+      child: Stack(children: [
+    Align(
+        alignment: Alignment.topRight,
+        child: ReportOrBlockMenuButton(l10n: l10n)),
+    Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: widgets,
+    )
+  ]));
+}
+
+Widget _createVacationBanner(BuildContext context) {
+  final AppLocalizations l10n = AppLocalizations.of(context)!;
+  final ThemeData theme = Theme.of(context);
+  return Container(
+      color: theme.colorScheme.primaryContainer,
+      height: 48.0,
+      child: Row(mainAxisAlignment: MainAxisAlignment.center, children: [
+        Wrap(
+          crossAxisAlignment: WrapCrossAlignment.center,
+          children: [
+            const Icon(
+              Icons.beach_access_outlined,
+              size: _inlineIconSize,
+            ),
```

Comment:
This Icon should be the same color as the text (onPrimaryContainer).

Reply:
good catch, thanks! changed other icon colors too